### PR TITLE
ssh-askpass-mac: submission

### PR DESCRIPTION
**Pull request by sstallion**
#### Description

New port submission for [ssh-askpass-mac](https://github.com/lukas-zronek/ssh-askpass-mac).

###### Type(s)

submission

###### Tested on

macOS 12.4 21F79 x86_64
Xcode 13.4.1 13F100

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by sstallion** [comment]
Ping!

File: aqua/ssh-askpass-mac/Portfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               xcode 1.0
+
+name                    ssh-askpass-mac
+github.setup            lukas-zronek ${name} 1.2 v
```
Comment (herbygillot) [inline aqua/ssh-askpass-mac/Portfile:8]: Using `${name}` in this way is not necessary.

`github.setup` implicity sets `${name}`, among other variables.

So you can delete the `${name}` line, and simply do:
```
github.setup            lukas-zronek ssh-askpass-mac 1.2 v
```

`${name}` will be set as expected.

Comment (sstallion) [reply aqua/ssh-askpass-mac/Portfile:8]: Will do!

**Comment by huyz** [comment]
@sstallion any update? ssh-askpass-mac has been at v3 for a while now

**Comment by huyz** [comment]
Btw, there's a typo `machports` at https://github.com/macports/macports-ports/pull/15399/files#diff-c9e89d623b72b87c372840043a024edea345a47937045ba3c778ac8041abed6fR41

**Pull request merged**